src/common/polyfills/prepend.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import PolyfillManager from './polyfill-manager';
+import getLogger from '../utils/logger';
+
+export default class PrependPolyfill {
+  static id: string = 'prepend';
+  static _logger: any = getLogger('PrependPolyfill');
+
+  static install(): void {
+    const parent = document.createElement('div');
+    if (parent.prepend) {
+      PrependPolyfill._logger.debug('No need to install polyfill');
+      return;
+    }
+    PrependPolyfill._logger.debug('Installing polyfill');
+    (function(arr) {
+      arr.forEach(function(item) {
+        if (item.hasOwnProperty('prepend')) {
```

Comment:
you already have a test here to tell you if it's installed or not, so seems redundant to do it before as well